package/src/contexts/messageInputContext/__tests__/MessageInputContext.test.tsx

```diff
@@ -0,0 +1,1173 @@
+import React, { PropsWithChildren } from 'react';
```

Comment:
This file is awfully long, please DRY where possible by using `it.each`, extracting large consts to external files and splitting where needed